documentation/concepts.md

```diff
@@ -169,6 +218,10 @@ consumer groups will be coming in the near future which will allow a consumer
 to pick up where it left off and provide fault-tolerant consumption of
 streams.
 
+```
+This ties back to the reporting worker starved to death, but clinging on to an *offset* to the bitter end not to loose probable state. When stateful consumer groups are implemented the reporting worker need not to be restarted with no state, but only to be initiated with it's state and it can resume from there.
```

Comment:
```suggestion
This ties back to the reporting worker starved to death, but clinging on to an *offset* to the bitter end so as not to lose probable state. When stateful consumer groups are implemented the reporting worker will not be restarted without state, but can resume from where it left off.
```

`Probable` in the sentence above doesn't make sense to me. Do you mean mutable or unsaved state? 🤔

Reply:
probable state - this is since there is the small but not insignificant event that the consumer has time time to complete the work, but not enough time to save its own state (assuming thats the order the consumer is implemented. Perhaps there is some atomic behavior one could use - but I do not know of it)